lib/utils/cleanup-html.js

```diff
@@ -42,7 +42,7 @@ function _removeMatchingAttributes ($html, selector, regex) {
 
 function cleanupHtml ($html, {
     remove = cleanupHtmlDefaults.remove,
-    removeNoText = cleanupHtmlDefaults.removeNoText,
+    // removeNoText = cleanupHtmlDefaults.removeNoText,
```

Comment:
Commited by accident?

Reply:
No. It is in the lint commit to make the lint pass, because the `removeNoText` below is also commented. I plan to remove this anyway, is coming from the v2 cleanup but not very useful because it can remove other elements inside.